turbinia/processors/mount_local.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Evidence processor to mount local images or disks."""
+
+import logging
+import os
+import subprocess
+import tempfile
+
+from turbinia import config
+from turbinia import TurbiniaException
+
+log = logging.getLogger('turbinia')
+
+def PreprocessMountDisk(evidence):
+  """Locally mounts disk in an instance.
+
+  Args:
+    evidence: A turbinia.evidence.RawDisk or subclass object.
```

Comment:
This is a bit unclear to me. Subclass of what?

Reply:
Updated to make it a little more clear.